turbo/src/main/kotlin/dev/hotwire/turbo/util/TurboUriHelper.kt

```diff
@@ -58,7 +58,11 @@ internal class TurboUriHelper(val context: Context) {
     private fun getContentUriAttributes(context: Context, uri: Uri): TurboUriAttributes? {
         val projection = arrayOf(OpenableColumns.DISPLAY_NAME, OpenableColumns.SIZE)
         val mimeType: String? = context.contentResolver.getType(uri)
-        val cursor = context.contentResolver.query(uri, projection, null, null, null)
+        val cursor = try {
+            context.contentResolver.query(uri, projection, null, null, null)
+        } catch (ignored: Throwable) {
+            null
+        }
```

Comment:
Can this throw an exception? The source documentation suggests it'll return `null` instead of an exception.

Reply:
Yeah, and I don't trust it, or, to be more precise, I don't trust Samsung that they didn't tinker with it 😅

Reply:
Ha, sounds good. I'm ok leaving it 👍